Source/Urho3D/Graphics/ModelView.cpp

```diff
@@ -1254,11 +1255,12 @@ SharedPtr<Model> ModelView::ExportModel(const ea::string& name) const
     return model;
 }
 
-StringVector ModelView::ExportMaterialList() const
+ResourceRefList ModelView::ExportMaterialList() const
 {
-    StringVector result;
+    ResourceRefList result;
+    result.type_ = Material::GetTypeStatic();
```

Comment:
you can use constructor instead
 explicit ResourceRefList(StringHash type)

Reply:
👍